app/models/backend_api.rb

```diff
@@ -5,6 +5,8 @@ class BackendApi < ApplicationRecord
   include SystemName
   include ProxyConfigAffectingChanges::ModelExtension
 
+  audited :allow_mass_assignment => true
```

Comment:
what is this for?

Reply:
One audit for all changes, not one per change. We use it everywhere.